packages/grafana-ui/src/components/FilterPill/FilterPill.story.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Story } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { FilterPill, FilterPillProps } from './FilterPill';
+import { withCenteredStory } from '@grafana/ui/src/utils/storybook/withCenteredStory';
+import mdx from './FilterPill.mdx';
+import { getAvailableIcons } from '../../types';
+
+export default {
+  title: 'General/FilterPill',
+  component: FilterPill,
+  decorators: [withCenteredStory],
+  argTypes: {
+    icon: { control: { type: 'select', options: getAvailableIcons() } },
```

Comment:
Nitpick: do we have any preference over usage of `getAvailableIcons` and `iconOptions` found [here](https://github.com/grafana/grafana/blob/master/packages/grafana-ui/src/utils/storybook/knobs.ts#L6-L14)? 

Button and Badge stories currently use `iconOptions` maybe whichever we settle on we put it somewhere for sharing across all stories?

Reply:
Didn't know we had `iconOptions`. I don't have any preferences but in this case the `None` option doesn't work because the component always has an icon by default, even tho `None` is selected in the dropdown.